gateway/1.6/modules/ROOT/pages/flex-local-configuration-reference-guide.adoc

```diff
@@ -238,7 +238,7 @@ spec:
 |`spec.targetRef.name`
 |Required
 |N/A
-|The API instance identifier to which the policy is bound to.
+|The API instance or upstream service identifier to which the policy is bound to.
```

Comment:
to which the policy is bound

or

which the policy is bound to

Reply:
Good catch! Thanks!